lib/flutter_webview_plugin.dart

```diff
@@ -61,22 +92,43 @@ class FlutterWebviewPlugin {
 
   /// Start the Webview with [url]
   /// - [withJavascript] enable Javascript or not for the Webview
+  ///     iOS WebView: Not implemented yet
+  ///     android: Implemented.
   /// - [clearCache] clear the cache of the Webview
-  /// - clearCookies] clear all cookies of the Webview
+  ///     iOS WebView: Not implemented yet
+  ///     iOS WKWebView: will implement later
+  ///     android: Implemented
+  /// - [clearCookies] clear all cookies of the Webview
+  ///     iOS WebView: Not implemented yet
+  ///     iOS WKWebView: will implement later
+  ///     android: Implemented
+  /// - [hidden] not show
+  ///     iOS WebView: not shown(addSubView) in ViewController
+  ///     android: Implemented
+  ///   [fullScreen]: show in full screen mode, default true
+  ///     iOS WebView: without rect, show in full screen mode
+  ///     android: Not implemented yet
```

Comment:
fullscreen is implemented on Android, but hidden is not

Reply:
OK